.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args:
+          - --line-length=79
+  - repo: https://gitlab.com/PyCQA/flake8
```

Comment:
instead of gitlab, this needs to be:

https://github.com/PyCQA/flake8

following: https://github.com/AICoE/aicoe-ci/issues/53